assembl/static2/js/app/pages/changePassword.jsx

```diff
@@ -75,14 +77,27 @@ class ChangePassword extends React.Component {
                   <Translate value="login.oldPasswordWarning" />
                 </span>
               </div>
-              <FormGroup className="margin-m">
+              <FormGroup className="margin-m align-flex">
                 <FormControl
                   type="password"
                   name="password1"
                   required
                   placeholder={I18n.t('login.newPassword')}
                   onChange={this.handleChangePassword}
                 />
+                <Helper
```

Comment:
This should become a component of its own, taking css classes, children nodes.

Reply:
I understand that putting 3 times the same code in different components is bad, but the helper is already a modularize component that I put around an input. Not sure what you want me to do.

Reply:
Already discussed, in person.